codelabs/Drafts/Archived_Module1_APITesting.md

```diff
@@ -0,0 +1,183 @@
+summary: Module 1 of the API Testing course. Module 1 will give you a high level understanding of what an API is and why testing APIs is important. This course will guide you on all you need to know to get started with continuous API testing.
+id: Module1-APITesting
+categories: beginner
+tags: zapi
+environments: Web
+status: One or more of (Draft, Published, Deprecated, Hidden)
+feedback link: https://forms.gle/CGu4QchgBxxWnNJK8
+analytics account: UA-86110990-1
+author:Lindsay Walker
+<!-- ------------------------ -->
+# Module 1 – API Testing Basics
+
+<!-- ------------------------ -->
+## 1.01 What is an API?
+Duration: 0:02:00
+
+APIs are the term used to describe web services that connect different platforms or data services together. An example is the Uber application. The way that you search, book, and pay are all done through APIs.
+
+**API** stands for **A**ppliation **P**rogramming **I**nterface, which is a set of definitions and protocols for how to communicate, with certain information being required from the client, and certain information being returned by the system that receives the request.
```

Comment:
## 1.01 What is an API?
Duration: 0:02:00

API is the term used to describe web services that connect different platforms or data services together. An example is the Uber application. The way that you search, book, and pay are all done through APIs.

**API** stands for **A**ppliation **P**rogramming **I**nterface, which is a set of definitions and protocols for how to communicate, with certain information being required from the client, and certain information being returned by the system that receives the request.